src/hotspot/os/linux/os_linux.cpp

```diff
@@ -4480,19 +4488,25 @@ void os::Linux::numa_init() {
   // bitmask when externally configured to run on all or fewer nodes.
 
   if (!Linux::libnuma_init()) {
+    if ((UseNUMA && FLAG_IS_CMDLINE(UseNUMA)) ||
+        (UseNUMAInterleaving && FLAG_IS_CMDLINE(UseNUMAInterleaving))) {
+      // Only issue a warning if the user explicitly asked for NUMA support
+      log_warning(os)("NUMA support is disabled as libnuma failed to initialize");
+    }
```

Comment:
Afaics this block can also use `disable_numa()`.

Reply:
I agree, addressed.